### PR TITLE
fix: Assign boolean to update-release explicitly

### PR DESCRIPTION
**Pull request by cagiti**
Assigning the boolean to the `--update-release` parameter explicitly.
Signed-off-by: Cai Cooper <caicooper82@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -279,7 +279,7 @@ release: clean build test-slow-integration linux # Release the binary
 	GITHUB_TOKEN=$(GITHUB_ACCESS_TOKEN) REV=$(REV) BRANCH=$(BRANCH) BUILDDATE=$(BUILD_DATE) GOVERSION=$(GO_VERSION) ROOTPACKAGE=$(ROOT_PACKAGE) VERSION=$(VERSION) goreleaser release --config=.goreleaser.yml --rm-dist
 	# Don't create a changelog for the distro
 	@if [[ -z "${DISTRO}" ]]; then \
-		./build/linux/jx step changelog --verbose --header-file docs/dev/changelog-header.md --version $(VERSION) --rev $(PULL_BASE_SHA) --output-markdown changelog.md --update-release false; \
+		./build/linux/jx step changelog --verbose --header-file docs/dev/changelog-header.md --version $(VERSION) --rev $(PULL_BASE_SHA) --output-markdown changelog.md --update-release=false; \
```
Comment (abayer) [inline Makefile]: Might as well just throw `=` on all the flags to be safe/clear.

**Comment by abayer** [comment]
/lgtm

We can add the remaining `=`s later, since they're non-ambiguous as is, I think.

File: Makefile (hunk 1 above)
Comment (cagiti) [reply Makefile]: done

**Pull request merged**